src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Represents a bye command which allows users to exit the program.
+ */
+public class ByeCommand extends Command {
```

Comment:
It seems you are using different classes for UI output, it's okay.
You can also do this in one single class lah

Reply:
I thought it would be better if the Ui class was in charge of all the messages to be printed (Single Responsibility Principle?)